src/pie_modules/taskmodules/re_text_classification_with_indices.py

```diff
@@ -427,12 +428,19 @@ def __init__(
         self.allow_discontinuous_text = allow_discontinuous_text
         self.handle_relations_with_same_arguments = handle_relations_with_same_arguments
         self.argument_type_whitelist: Optional[List[Tuple[str, str]]] = None
+        self.argument_and_relation_type_whitelist: Optional[Dict[str, Tuple[str, str]]] = None
 
         if argument_type_whitelist is not None:
             # hydra does not support tuples, so we got lists and need to convert them
             self.argument_type_whitelist = [
                 (types[0], types[1]) for types in argument_type_whitelist
             ]
+        if argument_and_relation_type_whitelist is not None:
```

Comment:
After thinking about this a bit more, I think it is a good idea to do the handling of `argument_and_relation_type_whitelist` before `argument_type_whitelist` and if `argument_type_whitelist` is not provided, set it based on the values of `argument_and_relation_type_whitelist`. What do you think?

Reply:
we could also do the other way around: remove entries not in ...and_relation_whitelist from the other

Reply:
hmm ok, this seems to be not straight forward und hard to communicate to the user, let's not change one of the whitelists depending on the other. sorry for the noise